pkg/updater/install.go

```diff
@@ -69,6 +69,9 @@ func (i *installer) installExperiment(pkg string, version string, image oci.Imag
 	if err != nil {
 		return fmt.Errorf("could not extract package layers: %w", err)
 	}
+	if pkg == "datadog-agent" {
+		service.ChownDDAgent(tmpDir)
+	}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Should we also do that on `installStable` for bootstrap?